api/v1alpha1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1alpha1 contains API Schema definitions for the kvrocks.com v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=kvrocks.com
+// +kubebuilder:object:generate=true
+// +groupName=kvrocks.com
```

Comment:
The formatting-related issues can be resolved by creating a new pull request specifically for them after configuring the CI.

Reply:
OK，I will remove it to make this pr clear.